### DISCUSSION TITLE
Ssh Certificate Authentication?

### DISCUSSION THREAD

**Discussion by accidentaldevelopment** [wezterm/wezterm#2982] [Q&A]
Hello! Are ssh certs expected to work for authentication. I found #1284 which seems to imply they should just work with the `libssh` backend, but I'm seeing a fallback to password auth (which is disabled on the server). Looks like the cert is never offered.

I can provide logs or open an issue if necessary, just wanted to know if this is even expected to work.

**Comment by wez** [comment]
Please open an issue; include the verbose output in it; eg: `wezterm ssh -v ...`

**Reply by accidentaldevelopment** [reply]
Created issue #2985. Thanks!
